src/core/utils/gamepad-handler.js

```diff
@@ -0,0 +1,100 @@
+export const btnIndexesToKeysMapping = {
```

Comment:
this file has been deleted in my last PR. It's now a library instead of a simple file.

Reply:
Ouai ben c'est relou y a trop de conflits

Reply:
Comment ça se fait alors que quand je pull master j'ai encore ce module ?

Faut faire une branche par feature et arrêter de pousser toutes ses modifs sur une branche qui ressemble à rien sinon on va pas s'en sortir.

Ou alors on repart de zéro